### ISSUE TITLE
same name files in different folders doesn't work

### ISSUE THREAD

**Issue by kazuhikoarase** [processing/p5.js-web-editor#1217]
#### Nature of issue?
- Found a bug

#### Details about the bug: 

- Web browser and version: Chrome
- Operating System: Windows
- Steps to reproduce this bug:

I create three assets as follows.
```
data.json
assets/data.json
assets/data2.json
```
https://editor.p5js.org/kazuhikoarase/sketches/dC8lEP02D

Then use loadStrings function and attempt to load them.
Without second assets 'assets/data.json', it works. But with it, causes error.
I saw that second assets's path(encoded) starts with 'assets/'.
Isn't is something related with the error?

Thank you.

```
file#0 = blob:https://editor.p5js.org/61b30737-c1cc-4d6c-a210-c0de8fba7567 
file#1 = assets/blob:https://editor.p5js.org/61b30737-c1cc-4d6c-a210-c0de8fba7567 
file#2 = blob:https://editor.p5js.org/4ed556dc-2c7d-4f04-a746-84edbf1f22c6 
> p5.js says: It looks like there was a problem loading your text file. Try checking if the file path [Error: [object ReadableStream]] is correct, or running a local server.[https://github.com/processing/p5.js/wiki/Local-server]
```

**Comment by kazuhikoarase** [comment]
Sorry, this is issue for p5.js.

**Comment by catarak** [comment]
Thanks for reporting! This is possibly related to #1205. I noticed that if you comment out the first line too, it still doesn't work.

**Comment by akshay-99** [comment]
Hey @catarak I would love to work on this. 

So the problem lies with the regex and replace() in resolveJSAndCSSLinks in PreviewFrames.jsx. If there are two files with the same name, even with different paths, the first pass would replace both their names in the original code the blob url of the first file. 

I will fix this and create a PR soon. 
Maybe I can fix #1205 too.
